### ISSUE TITLE
Can we disable the minimize button for the BrowserWindow?

### ISSUE THREAD

**Issue by lipis** [atom/electron#2970]
Like it's happening on the default About Box for example but for any window:

<img width="107" alt="screen shot 2015-10-03 at 00 07 48" src="https://cloud.githubusercontent.com/assets/125676/10258904/cec4d7f0-6962-11e5-8c07-4cf0c99ff117.png">

**Comment by alectic** [comment]
Try it this way

```JavaScript
var win = new BrowserWindow({ width: 800, height: 600, frame: false });
```

http://electron.atom.io/docs/v0.33.0/api/frameless-window/

**Comment by lipis** [comment]
@alectic it's not the same.. I still want to have the title and be able to close it.. and also don't forget windows :)

**Comment by alectic** [comment]
Yes you're right, coming from a tiling window manager where I close windows through key combinations, I didn't realize you still need to close the window. Unless you hide the frame completely and create a "virtual" frame yourself, add butons, style them and give them the functionality you need. But that will take away the native feel.

**Comment by lipis** [comment]
@alectic It will take the native look and throw it out of the freaking window :scream:. This is not an option :)

**Comment by ibocon** [comment]
I guess

>/atom/browser/web_view_guest_delegate.cc
manages the window's options.

>/atom/browser/native_window.cc
creates 'NativeWindow' object, which related with 'BrowserWindow'.

>/atom/browser/api/atom_api_window.h
This is the api for 'NativeWindow'

I hope this helps to find out a way.

**Comment by caracare521** [comment]
I have a workaround.....
First you just create a frameless window like what @alectic mentioned above.
Then, you can write some html to add a button (here you need to have someone to provide you an image), and use this button to trigger a browserWindow.close().

Tada, you have a window with only one close button now.

**Comment by lipis** [comment]
@caracare521 Now do that for every OS X, Windows and Linux versions that Electron currently supports it to make it look _natural_ :)

**Comment by zcbenz** [comment]
Fixed with #4156.

**Comment by lipis** [comment]
:100:
